rules/scanners-user-agents.data

```diff
@@ -215,3 +215,6 @@ Detectify
 # ZGrab scanner (Mozilla/5.0 zgrab/0.x)
 # https://zmap.io
 zgrab
+# Cyberscan
+# https://cyberscan.io
+zgrab
```

Comment:
What is the user-agent name of the Cyberscan scanner?
You added zgrab here, but I don't think it's zgrab, right? This seems to be a copy paste from the entry above.